tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,70 @@
+# Tracing OpenTelemetry
+
+[![Crates.io: tracing-opentelemetry](https://img.shields.io/crates/v/tracing-opentelemetry.svg)](https://crates.io/crates/tracing-opentelemetry)
+[![Documentation](https://docs.rs/tracing-opentelemetry/badge.svg)](https://docs.rs/tracing-opentelemetry)
+[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE.txt)
```

Comment:
I think this link should be changed to point at the LICENSE file at the root of this repo --- I _think_ this is now a broken link?

Reply:
yes. missed this one.